docs/content/how-to/common/multi-arch-on-hcp.md

```diff
@@ -0,0 +1,41 @@
+# Multi-arch on Hosted Control Planes
+## General
+Several platforms now support multi-arch capable HostedClusters when a multi-arch release image or stream is used for 
+the HostedCluster. This means a HostedCluster can manage NodePools with different CPU architectures. 
+
+!!! note
+
+    An individual NodePool only supports one CPU architecture and cannot support multiple CPU archiectures within the 
+    same NodePool.
```

Comment:
```suggestion
    An individual NodePool only supports one CPU architecture and cannot support multiple CPU architectures within the 
    same NodePool.
```

Reply:
I do have a PR that should catch things like this in the future 😅 - https://github.com/openshift/hypershift/pull/4700

Reply:
fixed